tests/terraform/checks/resource/aws/test_IMDSv1Disabled.py

```diff
@@ -0,0 +1,63 @@
+import unittest
```

Comment:
We may want to migrate to `pytest`, more modern.

Reply:
The build is using pytest runner, so it might work ootb